lizmap/modules/admin/templates/server_information.tpl

```diff
@@ -123,8 +123,9 @@
             {if $displayPluginActionColumn }
                 <th>{@admin.server.information.qgis.plugin.action@}</th>
             {/if}
-        <tr/>
+        </tr>
         {foreach $data['qgis_server_info']['plugins'] as $name=>$version}
+        {if $version['name']}
```

Comment:
The idea was to show that plugins **are** missing for WfsOutputExtension and AtlasPrint.
With this PR, we are not advertising possible plugins to enhance LWC anymore.
Either we let the server providing the name (next version available), or only L130, if instead `$version['name']` is not defined you should switch to the JSON key.

A fix has been made in the server side to always add the name.

Reply:
I add this because I have nothing in the first colonne and not found in the second one. So my fix could be: 
```
{if $version['name']}
<th style="width:20%;">{$version['name']}</th>
{else}
<th style="width:20%;">{$name</th>
{/if}
```

Reply:
> I add this because I have nothing in the first colonne

Yes, this is fixed in the next version of the plugin

> and not found in the second one

Yes, this is on purpose. So I can control from the plugin, easier to release than a LWC version

Your proposed fix looks good. But anyway, it's already fixed in the next version of the plugin. And the minimum lizmap server version required is hard-coded and upgraded for each LWC release now. So your `else` should never be called I think.

Let's close this PR, I will commit straight about the `<tr/>`. I should be alright.

Reply:
Do not close this PR, I'd like to add tests to this page.